src/main.c

```diff
@@ -1961,7 +1972,7 @@ static void cmdline_handler(int argc, char **argv)
 		if (bootslot)
 			r_context_conf()->bootslot = bootslot;
 		if (handlerextra)
```

Comment:
This condition should never evaluate to true
```suggestion
		if (handler_arrgs)
```

Reply:
OK.